### ISSUE TITLE
Warning when running in verbose mode

### ISSUE THREAD

**Issue by AlexChesters** [gillesdemey/node-record-lpcm16#32]
A warning is printed to the console when running in verbose mode.

```
(node:34135) Warning: No such label 'End Recording' for console.timeEnd()
```

**Comment by AlexChesters** [comment]
This is probably a result of https://github.com/nodejs/node/issues/6452

**Comment by gillesdemey** [comment]
Thanks for tracking this down, I was wondering how that would even be possible without a change in the API 😅

**Comment by AlexChesters** [comment]
No worries @gillesdemey, I've had a quick go at fixing it but I've not really got anywhere :(

**Comment by gillesdemey** [comment]
This will be fixed in `1.0.0` where we no longer use `console.time`
